src/WebJobs.Script.WebHost/Diagnostics/EventGenerator.cs

```diff
@@ -85,47 +88,47 @@ public void LogFunctionExecutionEvent(string executionId, string siteName, int c
             }
 
             [Event(65520, Level = EventLevel.Verbose, Channel = EventChannel.Operational)]
-            public void RaiseFunctionsEventVerbose(string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, string hostVersion, string eventCreatedAt)
+            public void RaiseFunctionsEventVerbose(string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, string hostVersion, string timeCreated)
```

Comment:
Regarding createdAt/dateTimeCreated I get your point that it's not just time but also date. However, all the existing columns PreciseTimeStamp/TimeStamp mention time only, so this is consistent.